### ISSUE TITLE
1.13 & 1.14 Issues with pathing

### ISSUE THREAD

**Issue by theOrakle** [itchannel/fordpass-ha#80]
I get the following on 1.13 & 1.14, but 1.12 works fine:
```
2021-02-03 11:53:36 DEBUG (MainThread) [custom_components.fordpass] username
2021-02-03 11:53:36 DEBUG (MainThread) [custom_components.fordpass] password
2021-02-03 11:53:36 DEBUG (MainThread) [custom_components.fordpass] vin
2021-02-03 11:53:36 DEBUG (MainThread) [custom_components.fordpass] region
2021-02-03 11:53:36 DEBUG (MainThread) [custom_components.fordpass] North America & Canada
2021-02-03 11:53:36 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] Succesfully fetched token Stage1
2021-02-03 11:53:36 WARNING (MainThread) [custom_components.fordpass] [Errno 2] No such file or directory: 'custom_components/fordpass/fordpass_token.txt'
2021-02-03 11:53:36 WARNING (MainThread) [custom_components.fordpass] Error communicating with FordPass for 1FASTTRUCK321GO
2021-02-03 11:53:36 ERROR (MainThread) [custom_components.fordpass] Error fetching fordpass data: Error communicating with FordPass for 1FASTTRUCK321GO
2021-02-03 11:53:36 DEBUG (MainThread) [custom_components.fordpass] Finished fetching fordpass data in 0.774 seconds
```

Did one slight tweak to "make the pain stop" in the fordpass_new.py file:
Alter 
`self.token_location = "custom_components/fordpass/fordpass_token.txt"`
to fully qualified path
`self.token_location = "/full/path/to/hass/config/custom_components/fordpass/fordpass_token.txt"`

And all is well.  I'm not sure if it is do to how I instantiate Hass or not, but this is the only thing that fails...

**Comment by itchannel** [comment]
@theOrakle thanks for reporting this bug. 
I added that variable just incase people need to change it. It is on my list to look at a better way such as getting the config directory path from HASS.

**Comment by Realpsudo** [comment]
This near ended me having to pull all my hair off, until I read this!
Thanks for the solution @theOrakle

**Comment by degrashopper** [comment]
@theOrakle could you test if the following path works for you? /config/custom_components/fordpass/fordpass_token.txt
This path should work for all HA setups.

**Comment by itchannel** [comment]
@degrashopper I'll test that path on a couple  of my instances and see if it works.

**Comment by craw169** [comment]
I could not get fordpass integration to work until i changed self.token_location to:
 /config/custom_components/fordpass/fordpass_token.txt
in fordpass_new.py then i restarted and re-added integration and now it is working fine.

**Comment by itchannel** [comment]
Fixed in 1.22 release.
